rpm/sapwmp.spec

```diff
@@ -127,5 +126,6 @@ fi
 %dir %{_prefix}/lib/supportconfig
 %dir %{_prefix}/lib/supportconfig/plugins
 %{_prefix}/lib/supportconfig/plugins/sapwmp
+%{_libexecdir}/sapwmp/scripts
```

Comment:
Missing `%dir`?

Reply:
I remove `%dir` on purpose, since this folder will be used for `sapwmp` package only. Without `%dir`, it will package all the files belong to this folder as well.

Reply:
Of course, you're right, my RPM-fu was weak here. I'll mark this as resolved.